chapter_post_and_database.asciidoc

```diff
@@ -1248,6 +1266,11 @@ def home_page(request):
     )
 ----
 ====
+// CSANAD:  on the 4th line:
+//          `item.text = request.POST["item_text"]`
+//          we accessed item_text using the get() method previously:
+//          `item.text = request.POST.get("item_text", "")`
```

Comment:
yes it's a bit janky.  you can see the request.POST.get is still in the codeblock lower down.